REMAINING.PUBLIC.md

```diff
@@ -0,0 +1,70 @@
+### Remaining public aliases
+
+annotation_reader
+cache.app_clearer
+$commandId
+debug.templating.engine.php
+profiler.request_matcher
+router
+security.access.decision_manager
+security.acl.cache
+security.acl.dbal.connection
+security.acl.provider
+security.encoder_factory
+security.firewall
+security.password_encoder
+session.handler
+session.storage
+session.storage.filesystem
+templating
+templating.loader
+translator
+twig.loader
+
+### Remaining public services
+
+cache.adapter.* <- pools are private by default when created by config so not a pb
+cache.app
+cache_clearer <- for the cache:clear command
+cache.global_clearer <- for the cache:pool:clear command to clear all pools
+cache.system
+cache_warmer <- required to bootstrap the kernel
+data_collector.cache
+data_collector.dump
```

Comment:
Do collectors actually need to be public ? AFAIK, they are needed only by the profiler, and so could become private